docs/Installation-Guide.rst

```diff
@@ -84,26 +84,71 @@ LightGBM uses **CMake** to build. Run the following commands:
 
 Also you may want to reed `gcc Tips <./gcc-Tips.rst>`__.
 
-OSX
-~~~
+macOS
+~~~~~
 
-LightGBM depends on **OpenMP** for compiling, which isn't supported by Apple Clang.
+LightGBM depends on **OpenMP** for compiling, which isn't supported by default Apple Clang.
 
-Please install **gcc/g++** by using the following commands:
+Make sure you have Apple's **Command Line Tools** installed
 
 .. code::
 
-  brew install cmake
-  brew install gcc --without-multilib
+   gcc
+   xcode-select --install
+
+Although the prompt message is a bit confusing, clicking **install** and
+following the instructions will install **Command Line Tools**, not *Xcode*
+
+
+Install GNU Compiler Collection, **gcc with OpenMP enabled**
 
-Then install LightGBM:
+Choose the binaries version number based on your OS `(for more info) <http://hpc.sourceforge.net>`_.
 
 .. code::
+   
+   High Sierra, Sierra & El Capitan : gcc-7.1-bin.tar.gz
+   Yosemite: gcc-5.1-bin.tar.gz
 
-  git clone --recursive https://github.com/Microsoft/LightGBM ; cd LightGBM
-  export CXX=g++-7 CC=gcc-7
-  mkdir build ; cd build
-  cmake ..
+.. code::
+   
+   cd Downloads
+   wget http://prdownloads.sourceforge.net/hpc/gcc-7.1-bin.tar.gz
+   # This will install the compiler in `/usr/local/bin`
+   sudo tar -zxvf gcc-7.1-bin.tar.gz -C/
+   # to remove the package from your Downloads directory
+   sudo rm gcc-7.1-bin.tar.gz
+   # to check the installation
+   gcc -v
+
+Install *`CMake <https://cmake.org/overview/>`_*
```

Comment:
Why not `brew install cmake`?

Reply:
I prefer to go directly with **cmake**
Personal experience:
In genomics, I've lost tremendous amount of time to debug when things were installed with *homebrew* or *macport*, now I stay far far away!